sample-app/README.md

```diff
@@ -131,20 +132,16 @@ fetching products from database: done
 
 ## Integration Tests
 
-The following script will bring up the docker-compose environment, run the
-curl commands above, verify the results, and bring down the environment:
+The following script will bring up the docker-compose environment, run the curl
+commands above, verify the results, and bring down the environment:
 
 ```shell-session
 ./run-tests.sh
 ```
 
-## Other Sample Apps
-
-- [hello-vault-go][hello-vault-go]
```

Comment:
Do we maybe want to have this list in both? Or do you think it's okay just to have it at the top level

Reply:
I think it's OK to have it just at the top level since that presents an overview of the repo and makes references to other similar repos. It's also consistent with the other 2 repos.

Reply:
okay sounds good!